### PR TITLE
Introduce TID hotfix for SecureBoot in sle12sp5

### PR DESCRIPTION
**Pull request by mloviska**
In order to handle booting issue described in [Booting JeOS 12SP5 fails after using UEFI and applying all updates](https://bugzilla.suse.com/show_bug.cgi?id=1188852)
we need to introduce a specific test case to adjust system configuration for sle12sp5 (JeOS) only.

- Verification run: https://openqa.suse.de/tests/6865133#step/verify_efi_mok/1

### CODE REVIEW

**Review by b10n1k — APPROVED** [review]
GTG

**Comment by b10n1k** [comment]
> | /~~~|~~~|\ /|
> | | __ | | / |
> |___ ___/ | | |

That was supposed to be ascii-art of LGTM :)

**Comment by mloviska** [comment]
> > | /~~~|~~~|\ /|
> > | | __ | | / |
> > |___ ___/ | | |
> 
> That was supposed to be ascii-art of LGTM :)

I can read it clearly, for sure it is there. :rofl:

File: tests/console/verify_efi_mok.pm [hunk 1]
```diff
@@ -185,7 +185,7 @@ sub run {
     # run fs check on ESP
     record_info "ESP", "Partition [$esp_details->{partition}], \nFilesystem [$esp_details->{fs}],\nMountPoint [$esp_details->{mount}]";
     assert_script_run "umount $esp_details->{mount}";
-    assert_script_run "fsck.vfat -tvV $esp_details->{partition}";
+    assert_script_run "fsck.vfat -tvVp $esp_details->{partition}";
```
Comment (Vogtinator) [inline tests/console/verify_efi_mok.pm:188]: Why `-p`? Shouldn't the test verify that it's clean?

File: tests/console/verify_efi_mok.pm [hunk 2]
```diff
@@ -149,7 +149,7 @@ sub get_esp_info {
         if (!defined($drive) && $line =~ /gpt/ && $line =~ /$vbd/) {
             ($drive) = split(/:/, $line, 2);
         }
-        if (!defined($esp_part_no) && $line =~ /boot,\s?esp/) {
+        if (!defined($esp_part_no) && $line =~ /boot,\s?esp|boot/) {
```
Comment (Vogtinator) [inline tests/console/verify_efi_mok.pm:152]: Could use a comment that this is due to ancient parted in 12 SP5 not detecting ESP types

File: tests/console/verify_efi_mok.pm (hunk 1 above)
Comment (mloviska) [reply tests/console/verify_efi_mok.pm:188]: It failed with https://openqa.suse.de/tests/6864292#step/verify_efi_mok/48

Comment (Vogtinator) [reply tests/console/verify_efi_mok.pm:188]: `-t  Mark unreadable clusters as bad.`

sounds like it shouldn't be used here (and won't really make sense on virtual HW anyway)

File: tests/console/verify_efi_mok.pm (hunk 2 above)
Comment (mloviska) [reply tests/console/verify_efi_mok.pm:154]: Comment added.

File: tests/console/verify_efi_mok.pm (hunk 1 above)
Comment (mloviska) [reply tests/console/verify_efi_mok.pm]: https://openqa.suse.de/tests/6869789#step/verify_efi_mok/48

**Review by Vogtinator — APPROVED** [review]

**Pull request merged**